lib/screens/survey/survey_screen.dart

```diff
@@ -0,0 +1,114 @@
+import 'package:flutter/material.dart';
+import 'package:survey_flutter/gen/assets.gen.dart';
+import 'package:survey_flutter/model/survey_model.dart';
+import 'package:survey_flutter/theme/app_constants.dart';
+import 'package:survey_flutter/theme/primary_button_style.dart';
+import 'package:survey_flutter/utils/build_context_ext.dart';
+
+const routePathSurveyScreen = '/survey';
+const _buttonHeight = 56.0;
+
+class SurveyScreen extends StatelessWidget {
+  final SurveyModel survey;
+  SurveyScreen({required this.survey, super.key});
```

Comment:
We usually put the `super.key` at the first
```suggestion
  SurveyScreen({ super.key, required this.survey,});
```